Sources/Sentry/SentryOptions.m

```diff
@@ -708,4 +709,136 @@ - (void)setEnablePreWarmedAppStartTracing:(BOOL)enablePreWarmedAppStartTracing
 
 #endif // SENTRY_UIKIT_AVAILABLE
 
+#if defined(DEGUG)
+- (NSString *)description
```

Comment:
`m`: What about using [`debugDescription`](https://developer.apple.com/documentation/objectivec/1418956-nsobject/1418703-debugdescription) instead?

Reply:
Yeah that would work 👍🏻